schemas/dab.draft.schema.json

```diff
@@ -755,23 +755,35 @@
                       "description": "Database object name"
                     },
                     "parameters": {
-                      "type": "object",
-                      "description": "Dictionary of parameters and their values",
-                      "patternProperties": {
-                        "^.*$": {
-                          "oneOf": [
-                            {
-                              "type": "boolean"
-                            },
-                            {
-                              "type": "string"
-                            },
-                            {
-                              "type": "number"
+                      "oneOf": [
+                        {
+                          "type": "object",
+                          "description": "Dictionary of parameters and their values (deprecated)",
+                          "patternProperties": {
+                            "^.*$": {
+                              "oneOf": [
+                                { "type": "boolean" },
+                                { "type": "string" },
+                                { "type": "number" }
+                              ]
                             }
-                          ]
+                          }
+                        },
+                        {
+                          "type": "array",
+                          "description": "Array of parameter objects with metadata",
+                          "items": {
+                            "type": "object",
+                            "required": ["name"],
+                            "properties": {
+                              "name": { "type": "string", "description": "Parameter name" },
+                              "required": { "type": "boolean", "description": "Is parameter required" },
+                              "default": { "type": ["string", "number", "boolean", "null"], "description": "Default value" },
+                              "description": { "type": "string", "description": "Parameter description" }
```

Comment:
given the format of accepting description for multiple parameters is a `,` - comma separated string, we should mention here that the description of individual parameters shouldn't contain `,`.